### ISSUE TITLE
WebNowPlaying Rainmeter Cleartext

### ISSUE THREAD

**Issue by TheGreyWizard** [ytmdesktop/ytmdesktop#360]
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**.
- [X] I verified that there is no open issue for the same subject.

**Describe the bug**
ClearText Rainmeter skin continuously seems to download now playing information from YTMDP through  WebNowPlaying extension. Problem disappears when YTMDP is terminated. Problem doesn't occur when now playing information from other webpages are fetched.

**To Reproduce**
1. Load 'ClearText' skin on rainmeter and set the player to 'WebNowPlaying'.
2. Open YTMDP and begin playing any song.
3. See network usage go up.

**Expected behavior**
Now playing information is fetched and displayed correctly on rainmeter skins with minimal network usage.

**Screenshots**
Network usage without YTMDP running.
![image](https://user-images.githubusercontent.com/42782192/92029625-1c920480-ed66-11ea-91bf-0441747af9ea.png)

Network usage with YTMDP running.
![image](https://user-images.githubusercontent.com/42782192/92030149-eacd6d80-ed66-11ea-8cd7-10bf58608ae6.png)
![image](https://user-images.githubusercontent.com/42782192/92030175-f6b92f80-ed66-11ea-952e-71774570b272.png)

**Environment (please complete the following information):**
 * YTMDesktop version: ***vx.x.x***
 * OS: ***Windows*** or ***Linux*** or ***Mac***
 * OS version: ***X***
 * Arch: ***x64*** or ***x86***
* Installation way: ***like .exe or snapcraft or another way***

**Comment by tjhrulz** [comment]
Since the cause of this has not been fixed yet in this app I have fixed the cause on my end of the WebNowPlaying plugin, so as long as people have version 0.5.0.0 they will not encounter this bug.

However I still strongly encourage you to add support for caching info in the app and only send the info as it changes as that was what caused this in the first place.

**Comment by Alipoodle** [comment]
This sounds to be the same issue as #283
TL;DR Sending to much.
This ticket will be closed in favour of the other ticket for fixing this issue.
